### ISSUE TITLE
Results of training with cityscapes

### ISSUE THREAD

**Issue by manuel-88** [hellochick/ICNet-tensorflow#44]
As far as I understood the inference results of the ICNet tensorflow model are from the already trained weights of the original paper. Did someone tried to reach the results of the original model with the provided trainingsscript? I tried different configurations but my scores are never higher than 60% on the cityscape val set (compared to the original model it should be 67.67%). I test different parameters, filter scales and tried it with and without batch normalization. I just wonder if this score of 67,67% is reachable?

**Comment by hellochick** [comment]
Hey @manuel-88 , I have not tried to train on the cityscapes dataset from scratch yet. But I think it might be reachable. Did you try to train with `--random-scale` and `--random-mirror`? Data augmentation would be very helpful. Also, you need to set `--filter-scale=2` to train on non-pruned model.

**Comment by manuel-88** [comment]
Yes, I used data augmentation and I also trained with filter scale 2 but the losses are after 60000 iterations 1/3 higher than on the unpruned model trained over the same iterations. That's strange for me because I thought more weights should give better results?
Is it possible that  they used for the Cityscape ICNet Model a pretrained PSPNet?

**Comment by hellochick** [comment]
@manuel-88, the non-pruned model would converge much slowly since there are more parameters need to train.
Yes, there is a way to use a pretrained PSPNet weights on ICNet model. I used their pretrained weights of PSPNet50 trained on ADE20k dataset, and keep training for the sub network of ICNet, thus get a good results!

**Comment by manuel-88** [comment]
ok thank you. I will try it
